functions.php

```diff
@@ -162,4 +162,16 @@ function successMessage (bool $successfulUpload) {
     }
 }
 
+/**
+ * updates the delete field of the about me table of the provided db to 1
+ *
+ * @param PDO $db PDO connects to the required database
+ *
+ * @param $id string value
```

Comment:
Need to describe what this param is and add a return void description

Reply:
done